app/controllers/characteristics_controller.rb

```diff
@@ -1,6 +1,26 @@
 class CharacteristicsController < ApplicationController
   load_and_authorize_resource
 
+  def new
+    render layout: "custom"
+  end
+
+  def create
+    @characteristic = Characteristic.new(params[:characteristic])
+    # @design_method.principle = ""
+
+    respond_to do |format|
+      if @characteristic.save
+        @citations
+        format.html { redirect_to @characteristic, notice: 'Characteristic was successfully created.' }
```

Comment:
See [here](https://github.com/TheDesignExchange/app/blob/83a2926664b4184935a48af95fe33c4fdc1a96cc/app/controllers/collections_controller.rb#L89) for an example of how you should handle redirecting the user back to the admin panel rather than characteristic pages when characteristics are successfully created/edited/deleted